tools/precision_sweep.py

```
@@ -0,0 +1,354 @@
+    return []
+
+
+def run_level(
+    grammar: Path,
+    level: int,
+    timeout: str,
+    max_tokens: str,
+    environment: dict[str, str],
+) -> Result:
+    started = time.monotonic()
+    completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)